client/css/style.css

```diff
@@ -775,6 +775,10 @@ kbd {
 	width: 100%;
 }
 
+.input:disabled {
```

Comment:
I think we should disable `:hover` on disabled elements (so our `.input:hover` should become `.input:not(:disabled):hover`, what do you think?